Nodejs/Product/Nodejs/CodeWindowManager.cs

```diff
@@ -20,10 +20,14 @@
 
 namespace Microsoft.NodejsTools {
     class CodeWindowManager : IVsCodeWindowManager, IVsCodeWindowEvents {
+#if FALSE
         private readonly IVsEditorAdaptersFactoryService _adapterService;
-        
```

Comment:
This property is only referenced in `#if FALSE` blocks. Rather than remove those as part of this change, I just move this field into the block as well. A separate, more targeted change can remove all the `#if FALSE` blocks.